doc/source/guidelines/PyMAPDL_WSL_Guide.rst

```diff
@@ -0,0 +1,423 @@
+  .. _ref_guide_wsl:
+
+
+ANSYS on Windows Subsystem for Linux and Docker
+###############################################
+
+This guide shows you how to use Ansys products, more specifically PyMAPDL, in the Windows Subsystem for Linux (WSL).
```

Comment:
```suggestion
This guide shows you how to use a PyAnsys library, more specifically PyMAPDL, in the Windows Subsystem for Linux (WSL).
```

Reply:
Given that this only a section of the dev guide, I've changed the terminology accordingly.